config/dataworks_schema.yml

```diff
@@ -175,6 +180,8 @@ components:
             - Valid
             - Withdrawn
             - Other
+        date_information:
```

Comment:
What is date_information? Please document.

Reply:
Added a comment